tests/rustdoc-json/allow_unsized.rs

```diff
@@ -0,0 +1,139 @@
+#![crate_name = "allow_unsized"]
+
+use std::fmt::Debug;
+use std::marker::PhantomData;
+
+pub trait CustomSized: Sized {}
+impl CustomSized for u8 {}
+
+// Generic functions
+//@ is "$.index[?(@.name=='func_custom')].inner.function.generics.params[0].kind.type.allow_unsized" false
+pub fn func_custom<T: ?Sized + CustomSized>() {}
```

Comment:
This should test that we don't remove the `?Sized` bound from the bounds list, even though it ends up not mattering.